create/creating-applications-with-cicd-pipelines.adoc

```diff
@@ -55,6 +55,12 @@ include::modules/op-mirroring-images-to-run-pipelines-in-restricted-environment.
 
 include::modules/op-running-a-pipeline.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
```

Comment:
Needs an anchor ID. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies